www/_template/concepts/build-pipeline.md

```diff
@@ -3,30 +3,44 @@ layout: layouts/content.njk
 title: The Build Pipeline
 description: Snowpack Build creates a production-ready website with or without a bundler
 ---
+THe Snowpack build pipeline runs on the command `snowpack build`. Snowpack then creates a production-ready website, with or without a bundler.
 
-![build output example](/img/snowpack-build-example.png)
-
-`snowpack build` - When you're ready to deploy your application, run the build command to generate a static production build of your site. Building is tightly integrated with your dev setup so that you are guaranteed to get a near-exact copy of the same code that you saw during development.
+This article will cover
+- What Snowpack does by default
+- When and how to use a bundler
+- Other options
```

Comment:
👍🏻 Like these summaries you’ve been adding to pages

Reply:
+1!

Reply:
what does "other options" mean though? this line could use some extra detail